Makefile

```diff
@@ -13,15 +22,18 @@ all: install
 
 build-linux:
 	@echo Build Linux amd64
-	env GOOS=linux GOARCH=amd64 $(GO) install -mod=readonly -trimpath ./...
+	env GOOS=linux GOARCH=amd64 $(GO) build -o $(COORDINATOR) ./cmd/coordinator
```

Comment:
Any reason the trimpath and -mod=readonly flags are removed ?

Reply:
No specific reasons though, thought they are not needed.